### ISSUE TITLE
schedule needs the ability to run a shell script

### ISSUE THREAD

**Issue by wizdude** [SpaceCP/Bugs#61]
one thing I currently do by hand is backup my server world/\* folder on a daily basis (or when I remember to do this).

a schedule seems to be a logical place to do this, however there is no "backup world" option and no ability to call a shell script or similar.

this would be great, since you could even go a step further and have a "world backup/restore" option/wizard which asks a few questions and then creates a scheduled task for you. the ability to edit this schedule is obviously missing at this time as well.

**Comment by JamyDev** [comment]
Allowing to set an arbitrarily executable file might not be super secure. Indeed this seems like a rather useful feature, however I won't implement it before we have some sort of container system. (Docker, namespaces, you name it)

**Comment by wizdude** [comment]
i agree - this probably wouldn't be a good solution, but perhaps could be implemented with scripts that live in a specific folder and enabled via a whitelisting configuration.

as an alternative, what I was initially trying to achieve here was making a backup of the world. perhaps that could be added as a feature while a more solid/secure solution such as containers has been developed.

**Comment by Willsr71** [comment]
Bumping this because I have a python backup script that I would like to use.
